platform/src/main/java/org/stellar/anchor/platform/callback/RestRateIntegration.java

```diff
@@ -110,10 +110,6 @@ void validateRateResponse(GetRateRequest request, GetRateResponse getRateRespons
     }
 
     if (request.getType() == GetRateRequest.Type.FIRM) {
-      if (Objects.requireNonNull(rate).getFee() == null) {
-        logErrorAndThrow(
-            "'rate.fee' is missing in the GET /rate response", ServerErrorException.class);
-      }
```

Comment:
Do we want to make the fee object optional?

Reply:
Its required in the SEP-38 responses

Reply:
Do we assume `fee` is necessary in a quote?